ServerAdministration/PractiaclDocker.md

```diff
@@ -0,0 +1,139 @@
+# Practical Docker in a swarm
+
+In the following the installation and setup process of DockerCE and Docker Swarm is described and at the end some examples on how to use Docker Services is given.
+
+## Installation of Docker 
+
+To install DockerCE on a node in the Swarm the following command can be used to setup the local Docker envionment:
+
+```bash
+curl -fsSL https://get.docker.com -o get-docker.sh | sh get-docker.sh
+```
+
+Once the Docker engine has been installed it is recomended that the user is added to the Docker group so every Docker command is not run as root, use the following command to do so:
+
+```bash
+sudo usermod -aG docker <USERNAME>@student.aau.dk
+```
+
+If the Docker installation is going to be used in a Docker Swarm at AAU the following changes is need since the AAU network is running on the default Docker subnet.
```

Comment:
```suggestion
If the Docker installation is going to be used in a Docker Swarm at AAU, the following changes are needed since the AAU network is running on the default Docker subnet.
```